src/components/checkbox/index.js

```diff
@@ -80,15 +80,22 @@ const Label = glamorous.span({
   }
 });
 
-const Checkbox = ({ children }) => (
+const Checkbox = ({ children, defaultChecked, disabled }) => (
   <MultipleChoice>
-    <Input type="checkbox" />
+    <Input type="checkbox" checked={defaultChecked} disabled={disabled} />
```

Comment:
where we are using inputs, we want to pass quite a few different props straight down to the Input component.

Rather than naming each one, we can instead name the props that we don't want to pass down (children) and then pass down ...rest.

So:

```
const Checkbox = ({ children, ...props }) => (
    <MultipleChoice>
     <Input type="checkbox" ...props} />
     <Label>{children}</Label>
    </MultipleChoice>
```